python/tvm/relay/backend/graph_runtime_codegen.py

```diff
@@ -36,15 +36,15 @@
 from __future__ import absolute_import
 
 from tvm.ndarray import empty
-from tvm.relay import build_module
+from tvm.relay import _build_module
```

Comment:
Why rename it to `_build_module`?

Reply:
The original way just won't work. tvm.relay.build_module does have the _GraphRuntimeCodegen interface registered but the underlying functions are not there. Change that to tvm.relay._build_module solves the problem. The python GraphRuntimeCodegen class is not used in any other modules so it might not been tested fully.